app/models/session.rb

```diff
@@ -1,5 +1,9 @@
 class Session < ApplicationRecord
   belongs_to :user
+  belongs_to :active_impersonator_session,
+    -> { where(status: :in_progress) },
+    class_name: "ImpersonationSession",
+    optional: true
```

Comment:
This will only be set for the `super_admin` session.  It represents the impersonation session that the Admin is _currently_ viewing.